parl/utils/replay_memory.py

```diff
@@ -46,6 +46,22 @@ def sample_batch(self, batch_size):
         terminal = self.terminal[batch_idx]
         return obs, action, reward, next_obs, terminal
 
+    def make_index(self, batch_size):
+        # index mapping to avoid sampling saving example
+        batch_idx = np.random.randint(
+            self._curr_size - 300 - 1, size=batch_size)
+        batch_idx = (self._curr_pos + 300 + batch_idx) % self._curr_size
```

Comment:
I understand that this function is used to generate some `safe` index for sampling, as in the DQN RPM, we used multiple continual images to generate `states` and `next_states`. However, we have stored `states` and `next_states` separately in this implementation. Why do we have to map the index again here?

Reply:
This step is not theoretically needed, I will remove it and verify it